msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -41,7 +41,7 @@ To add tabs to an existing meeting:
 1. In your calendar, select a meeting to which you want to add a tab.
 1. Select the **Details** tab and select <img src="~/assets/images/apps-in-meetings/plusbutton.png" alt="Plus button" width="30"/>. The tab gallery appears.
```

Comment:
This `plusbutton.png` is an inline icon but is large in size. Let's limit the size of inline icons to at max. 32x32. Ideally, 16x16 pixels is good. Only some icons that are not very clear at that low resolution can be used up to 32x32, though this resolution is not recommended.

Reason for size: line spacing gets awkward when using larger icons inline. See this in the article: 
![image](https://user-images.githubusercontent.com/49870256/157885385-b0c256a5-32fc-4fe6-ac08-ef491b1ee2cd.png)

Other guidance for inline icons are:

- To use the icons with transparency.
- Use type = icon as described in https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=main#icons

CC @v-silakshmi , @surbhigupta12 .